judgments/utils.py

```diff
@@ -51,6 +51,14 @@ def perform_advanced_search(
     return SearchResults.create_from_string(multipart_data.parts[0].text)
 
 
+def preprocess_query(query):
+    return remove_unquoted_stop_words(normalise_quotes(query))
+
+
+def normalise_quotes(query):
+    return re.sub(r"[“”]", '"', query)
```

Comment:
Is it also worth normalising single quotes?

Reply:
aha, been pondering  that - marklogic doesn't currently treat them as phrase delimiters, so it's probably less important. That is a further, thornier question - if we were to treat them as phrase delimiters, we need to be very careful about single quotes that are apostrophes (which i think means we're out of regex land and into chomsky grammars and such as we need to find matched pairs of apostrophes, not just blindly replace them), so i'd be inclined to leave it for now, maybe add a ticket for further investigation?